lake/ztests/use.yaml

```diff
@@ -20,14 +20,16 @@ outputs:
     data: |
       ===
       Switched to branch "main" on pool "POOL"
-      HEAD at POOL@main
       ===
       Switched to branch "b1" on pool "POOL"
       ===
       Switched to branch "b2" on pool "POOL"
-
   - name: stderr
     data: |
       default pool and branch unset
       default pool unset
       "branch": branch not found
+  - name: no-args-use.txt
+    regexp: |
+      HEAD at POOL@main
+      Lake: file.*/test
```

Comment:
I think it looks funny to use both "at" and ":".

Reply:
I don't think either is right but I'll go your way for now.